.goreleaser.yaml

```
@@ -25,26 +27,21 @@ builds:
     goarch:
       - amd64
       - arm64
-checksum:
-  name_template: 'checksums.txt'
-snapshot:
-  name_template: "{{ .Version }}-next-{{ .ShortCommit }}"
-changelog:
-  sort: asc
-  filters:
-    exclude:
-      - docs
-      - test
-      - goreleaser
+# Create a draft release in GitHub. We can write release notes manually.
+draft: true
 brews:
-  - tap:
+  - name: getenvoy
+    tap:
       owner: tetratelabs
       name: homebrew-getenvoy
       # Use token other than the default one (which is limited in scope to
       # https://github.com/tetratelabs/getenvoy only)
       token: "{{ .Env.HOMEBREW_GETENVOY_GITHUB_TOKEN }}"
     folder: Formula
     homepage: https://github.com/tetratelabs/getenvoy
-    description: Distribute, deploy and operate a fleet of Envoys
+    description: Download and run Envoy
```

nit this is not in line with the "install" command 😄 
```suggestion
    description: Install and run Envoy
```